pkg/finder/finder_test.go

```diff
@@ -2,10 +2,11 @@ package finder
 
 import (
 	"fmt"
-	"github.com/Boeing/config-file-validator/pkg/misc"
 	"path/filepath"
 	"testing"
 
+	"github.com/Boeing/config-file-validator/pkg/misc"
+
```

Comment:
import should be grouped altogether, so it's ok but you shouldn't have this empty line